model/types.go

```diff
@@ -161,3 +161,23 @@ func (s ERC20Storage) TableName() string {
 func ERC20ContractTableName(address []byte) string {
 	return "erc20_" + hexutil.Encode(address)
 }
+
+// ERC20Transfer represents the transfer event in erc20
+type ERC20Transfer struct {
+	Address     []byte `gorm:"-"`
+	BlockNumber int64  `gorm:"size:8;index"`
+	TxHash      []byte `gorm:"size:32;index"`
+	From        []byte `gorm:"size:20;index"`
+	To          []byte `gorm:"size:20;index"`
+	Value       string `gorm:"size:32"`
+}
+
+// TableName retruns the table name of this erc20 contract
+func (s ERC20Transfer) TableName() string {
```

Comment:
How about introduce `ContractName`?

Reply:
OK. I will add the first 3 char of contract name

Reply:
It will make our code more complex while query or insert data.